gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -192,7 +192,7 @@ def get_network_function(self, context, network_function_id):
     def consumer_ptg_added_notification(self, context, network_function_id,
                                         policy_target_group):
         cctxt = self.client.prepare(version=self.RPC_API_VERSION)
-        cctxt.cast(context,
+        return cctxt.call(context,
```

Comment:
This call can timeout and raise exception. Is there a top level exception handler to take care of this? or it is not needed?